test/boulder-tools/setup-govulncheck.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -feuxo pipefail
+
+# Implements a filesystem structure that can serve as an API for govulncheck.
+# See https://go.dev/security/vuln/database for more information.
+#
+# TODO(@pgporada) Remove this if we decide that the boulder-tools container can
+# have network access.
+
+GVCDIR="/tmp/govulncheck-local-database"
+if [ -d "${GVCDIR}" ]; then
+    rm -rf "${GVCDIR}"
+fi
+
+mkdir -p "${GVCDIR}"
+cd "${GVCDIR}"
+
+# Check out only the data we need from this repository.
+git clone -n \
+	--depth=1 \
+	--filter=tree:0 \
+	https://github.com/golang/vulndb .
+git sparse-checkout set --no-cone data/osv
```

Comment:
I don't think we need --no-cone (it's [discouraged](https://git-scm.com/docs/git-sparse-checkout#_internalsnon_cone_problems) by the git docs, and we just need this one directory, not all paths which match this pattern).

I think we _can_ set --sparse-index, since we're going to be deleting the index anyway, so there's no point in populating a full one.